core-bundle/src/Resources/contao/library/Contao/Dbafs.php

```diff
@@ -494,9 +494,17 @@ public static function syncFiles()
 		}
 
 		$objDatabase = Database::getInstance();
+		$objDatabaseEngine = $objDatabase
+			->prepare("SELECT ENGINE FROM information_schema.TABLES WHERE TABLE_NAME = 'tl_files' AND TABLE_SCHEMA = ?")
```

Comment:
Is this query really necessary? The Doctrine connection has a default table option named "engine", which we could check.

Reply:
Imho we don't need the default but the one from the actual table. Maybe Doctrine can help with this nevertheless?